language/move-vm/types/src/natives/function.rs

```diff
@@ -52,6 +52,17 @@ pub trait NativeContext {
     fn is_resource(&self, ty: &Type) -> bool;
 }
 
+/// The return value(s) of a native function
```

Comment:
Maybe add a comment here to mention that we did this intentionaaly to avoid allocation overhead of vectors, and suggests using smallvec later if necessary?

Reply:
Will do.